go.mod

```diff
@@ -154,3 +163,5 @@ require (
 	google.golang.org/grpc v1.54.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1
 )
+
+replace github.com/jimsmart/schema => github.com/semiotic-ai/schema v0.0.0-20230727171059-24e630184512
```

Comment:
Could you show me in the sink code PR above where this fork is in application.

Reply:
db.go, lines: 103, 139, 185

Reply:
Basically, the `Tables` and `PrimaryKeys` functions trigger a query that look for specific tables depending on which database driver is being used. It already has mysql, mssql, postgres, etc I just added clickhouse